### DISCUSSION TITLE
How to hide Header Underline in Markdown view?

### DISCUSSION THREAD

**Discussion by lexterror** [gsantner/markor#1386] [Q&A] [answered]
I tried setting the underline color to Alpha 0! I also tried "text-decoration:none" Please let me know if there is currently a way to do this!

Thank you!

**Comment by gsantner** [comment, marked as answer]
Inject this via Markor "inject" settings option:

`<style>h1,h2,h3,h4 { border-bottom: 0; } </style>`

you may also use the export -> html plaintext option, where you can see the injected css (shows what markor does predefine)

**Comment by lexterror** [comment]
Thank you so much, I thought it was underline!

**Comment by anisurrahman072** [comment]
Is there any way that I want only a specific # (header) as not underlined? I mean I want underline for all # but a specific # will have no underline
